### ISSUE TITLE
ADT composition video link timestamp typo

### ISSUE THREAD

**Issue by milesfrain** [JordanMartinez/purescript-jordans-reference#402]
## Context
```
Section:
  FP Philosophical Foundations

Issue Type:
  Bug

Folder/File(s):
  - FP Philosophical Foundations/Composition Everywhere.md
```

https://github.com/JordanMartinez/purescript-jordans-reference/blob/development/01-FP-Philosophical-Foundations/01-Composition-Everywhere.md#composing-types-algebraically

## Body

```
'Algebraic Data Types' as "Composable Data Types" (stop at 12:40)
```

The link's start time is 19:15, which is after the suggested stop time.

**Comment by JordanMartinez** [comment]
I think that's how long you're supposed to watch the video...? Maybe YouTube changed something? Who knows...

Anyways, I rewatched that part and found that `29:26` is a good place to stop.

**Comment by JordanMartinez** [comment]
Fixed in 7961e79cf5789d6d47711fec70398dcbe7f5e166

This issue will be closed in next release
